src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -26,6 +26,11 @@
           </button>
 
         </sky-dropdown-item>
+        <sky-dropdown-item *ngIf="searchResults.length === 0">
+          <div class="sky-autocomplete-no-results-found">
```

Comment:
We have a few CSS classes we can use for the padding. Mind replacing the `sky-autocomplete-no-results-found` class with `sky-padding-squish-default`? You can also remove the CSS styles from the stylesheet.